frontend/uddhar-frontend/src/coordinator/components/DisasterInput.jsx

```
@@ -0,0 +1,190 @@
+            />
+            <p className="text-right text-gray-500">
+                {wordCount}/200 characters
+            </p>
+            {errors.description && (
+              <p className="text-red-500 text-sm">{errors.description}</p>
+            )}
+          </div>
+
+          {/* location of the disaster */}
+          <div className="mb-20">
+            <label className="block text-gray-700">Location</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)